ComfortZoneApp/app/src/main/java/com/example/comfortzone/models/WeatherData.java

```diff
@@ -35,7 +35,7 @@ public double getTemp() {
     }
 
     public void setDate() {
-        DateTimeFormatter dtf = DateTimeFormatter.ofLocalizedDate(FormatStyle.SHORT);
+        DateTimeFormatter dtf = DateTimeFormatter.ofLocalizedDate(FormatStyle.FULL);
```

Comment:
see `SimpleDateFormat` if you need more customization: https://developer.android.com/reference/java/text/SimpleDateFormat